doc/changelog.md

```diff
@@ -1,151 +1,135 @@
+  `catalyst.cond`).
+
+  This feature is currently opt-in and experimental, and requires setting the
+  `autograph=True` flag in the `qjit` decorator:
 
   ```python
+  dev = qml.device("lightning.qubit", wires=1)
+
   @qjit(autograph=True)
-  @qml.qnode(qml.device("lightning.qubit", wires=n))
+  @qml.qnode(dev)
   def f(x):
-
-    for i in range(n):
-      qml.Hadamard(i)
-
       if x < 0.5:
-        y = jnp.sin(x)
+          qml.RY(jnp.sin(x), wires=0)
       else:
-        y = jnp.cos(x)
-
-      qml.RY(y, wires=i)
+          qml.RX(jnp.cos(x), wires=0)
 
-    return qml.probs()
+      return qml.expval(qml.PauliZ(0))
   ```
 
-  , which previously would be expressed as:
+  This feature is based on the AutoGraph module from TensorFlow, and
+  requires a working TensorFlow installation be available. In addition,
+  Python loops (`for` and `while`) are not currently supported, and do
+  not work in Autograph mode.
 
-  ```python
-  @qjit
-  @qml.qnode(qml.device("lightning.qubit", wires=n))
-  def f(x):
-
-    @for_loop(0, n, 1)
-    def repeat(i):
-      qml.Hadamard(i)
-
-      @cond(x < 0.5)
-      def correction():
-        return jnp.sin(x)
-
-      @correction.otherwise()
-      def correction():
-        return jnp.cos(x)
+  Note that there are some caveats when using this feature especially around
+  modifying global variables or object mutation inside of methods. A
+  functional style is always recommended when using `qjit` or AutoGraph.
 
-      y = correction()
-      qml.RY(y, wires=i)
+* The quantum adjoint operation can now be natively represented in Catalyst programs
+  using `catalyst.adjoint`.
+  [(#220)](https://github.com/PennyLaneAI/catalyst/pull/220)
 
-    repeat()
+  The `catalyst.adjoint` function computes the
+  adjoint of the quantum computation defined by a provided quantum function.
 
-    return qml.probs()
-  ```
+  An advantage of using the Catalyst-native adjoint is that the adjoint
+  operation now shares the usual Catalyst benefits; namely,
 
-  The feature is currently opt-in and requires setting the ``autograph=True`` flag in the ``qjit``
-  decorator. Note that there are some caveats when using this feature, especially around modifying
-  global variables or object mutation inside of methods. A functional style is always recommended
-  when using qjit or AutoGraph. Please see the documentation page on the [AutoGraph feature]() for
-  more details.
+  - reduced circuit size(and corresponding faster compile times!) when using
+    control flow, and
+  - performance benefits from Catalyst's Just-In-Time compilation.
 
-* Add support for natively representing the quantum adjoint operation. ``catalyst.adjoint`` computes the
-  adjoint of the quantum computation defined by a provided quantum function. For example:
+  Quantum functions with arbitrary Catalyst control flow can have their
+  adjoint versions computed, even when the control flow contains classical
+  preprocessing:
 
   ```python

```

Comment:
This is a very big example, I wonder if we could make it shorter.

Reply:
Yeah good point

Reply:
Done [c50dabf](https://github.com/PennyLaneAI/catalyst/pull/266/commits/c50dabf15614b8afd8329a24ae6f2ddbb839c7cb)